src/chrome/content/rules/Tim.it.xml

```diff
@@ -0,0 +1,21 @@
+<ruleset name="Tim.it (partial)">
+	<target host="tim.it" />
```

Comment:
It seems like this base host has an error connecting to HTTPS. Could you double check to see if HTTPS is available?

Reply:
Seems flaky at the moment:
```
$ curl -I https://tim.it
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to tim.it:443 
$ curl -I https://tim.it
HTTP/1.1 301 Moved Permanently
[...]
```